### ISSUE TITLE
different case "NewLines" and "Newlines" in similar assertions

### ISSUE THREAD

**Issue by Bananeweizen** [assertj/assertj#2754]
There are multiple assertions on CharSequence that allow ignoring or normalizing newlines. Unfortunately those methods differ in their writing of "newline" as part of the method name. E.g. when editing code to switch between these 2 methods, it is not sufficient to exchange `Ignoring` by `Normalizing`, you rather also have to switch between uppercase and lowercase `L`:

[isEqualToIgnoringNew**L**ines](https://www.javadoc.io/static/org.assertj/assertj-core/3.23.1/org/assertj/core/api/AbstractCharSequenceAssert.html#isEqualToIgnoringNewLines(java.lang.CharSequence))
[isEqualToNormalizingNew**l**ines](https://www.javadoc.io/static/org.assertj/assertj-core/3.23.1/org/assertj/core/api/AbstractCharSequenceAssert.html#isEqualToNormalizingNewlines(java.lang.CharSequence))

It seems reasonable to deprecate and rename all methods containing `NewLines` to instead use the more common `Newlines`. (I have not checked if the same problem appears outside CharSequence assertions)

* assertj core version: 3.23.1
* no test case

**Comment by joel-costigliola** [comment]
fair enough, will keep the existing assertion for backward compatibility

**Comment by icod** [comment]
Hi, can I work on this one?

**Comment by scordio** [comment]
Sure and thanks, @icod!

**Comment by icod** [comment]
Cool! 

A few questions:

- Should I use the 'since' parameter in the `@Deprecated` annotation? And if so, which version should I use? 
- What to do which the Javadoc of the deprecated public methods? The new method will have the correct Javadoc ('Newlines'). The old one can just have no Javadoc other than a `@deprecated Please use... instead.`, right? 
- Should there still be unit tests for the deprecated public methods?
